test/unit/read_preference.test.js

```diff
@@ -0,0 +1,138 @@
+'use strict';
+
+const { ReadPreference } = require('../../src');
+const chai = require('chai');
+chai.use(require('chai-subset'));
+const expect = chai.expect;
```

Comment:
```suggestion
import { ReadPreference } from '../../src';
import { expect } from 'chai';
```

If it doesn't give you too much trouble we can use TS here, just more drive by improvements.

Reply:
Done!